### ISSUE TITLE
Models predict always return Violence 

### ISSUE THREAD

**Issue by ducMNSD** [JimLee1996/AVSS2019#12]
Thank you for your greate work !
I have a question that when I use your pretrained model to transfer with my custom dataset, I have a problem: the predicts results always return Violence. 
My datasets have 4000(2000 fi - 2000 no), I use 20% for validation 
Validation accuracy is 0.911
